src/content/guides/compatible-devices.mdoc

```diff
@@ -43,24 +48,33 @@ The Centrapay service is supported on a range of payment and point of sale devic
 - M440
 - P400 (P400 PLUS)
 - P630
+- UX700 (Unattended Android)
 - V200T
 - V205C
 - V210
 - V240M
 - V400
 - V400C
 - VX690
-- UX700 (Unattended Android)
 
-## Windcave
+### Windcave
 - CHU200M
 - CHU200MP
 - CHU200T
 - CHU200TL
 - CHU200TP
 - IPP350
-- IWL25C
 - IWL252
+- IWL25C
 - LANE3000
 - MOVE5000
 - MTM300
+
+## E-commerce Integrations
+
+Centrapay supports online payments through several e-commerce options.
+If you're building your own website or using a custom platform, follow our [e-commerce guide](/guides/ecommerce-website) to integrate Centrapay directly.
+Alternatively, Centrapay is available on the following platforms:
+
+- [Shopify](https://www.shopify.com/)
```

Comment:
Do we or shopify/blackpepper have any material on how to enable Centrapay for eCommerce?

Reply:
@IsaacMurtagh not that I'm aware of. I know Farmlands have some material for how to enable Centrapay using Shopify but otherwise no. @alexBinns do you know of any?